subsquid/src/processor.ts

```diff
@@ -145,4 +145,11 @@ processor.addEventHandler(
 
 processor.addEventHandler("Oracle.PriceChanged", processOraclePriceChanged);
 
+processor.addEventHandler("*", async (ctx) => {
+  const signer = ctx?.event?.extrinsic?.signature?.address;
```

Comment:
@andresmechali `ctx?.event?.extrinsic?.signature?.address` i have added a fallback case in `saveEvent` however we need to identify what causes this be different in different cases